### ISSUE TITLE
New sign up UI

### ISSUE THREAD

**Issue by jd2rogers2** [Nonprofit-Exchange-Hub/web-app#278]
[mocks](https://www.figma.com/file/K7DT5jYcLKRc9epmNrO0N7/UX---Dev-Handoff?node-id=1%3A5&t=EJp3Re22jkzX0DEQ-0)
[image assets will be here](https://drive.google.com/drive/folders/17TyKrqROm9epjrv_cQvvQwNmniNInkq5?usp=share_link)

make sure to build off of https://github.com/Nonprofit-Exchange-Hub/web-app/pull/272

expanding off of the existing sign up page (`client/src/views/SignupCitizen.tsx`) please style and add form inputs according the mock above
- possible tabbing solution could be similar to existing in `client/src/components/Users/Auth/SignUpUserAndNonprofit/SignUpUserAndNonprofit.tsx`
- does not need to make an api request, functionality will be handled in separate ticket

**Comment by garimacha** [comment]
@jonathanchuangcodes this seems to be blocked by another PR. Ryan is working on that PR. Can you sycn with @jd2rogers2  to get another ticket to work on until 272 is cleared?

**Comment by garimacha** [comment]
@jonathanchuangcodes do you have the open branch for this ticket that you are working on?
